pkg/redis/mocks/Client.go

```diff
@@ -253,6 +253,27 @@ func (_m *Client) HSet(_a0 string, _a1 string, _a2 interface{}) error {
 	return r0
 }
 
+// HSetNX provides a mock function with given fields: key, field, value
+func (_m *Client) HSetNX(key string, field string, value interface{}) (bool, error) {
```

Comment:
Maybe write a test in pkg/redis/redis_client_test.go ?

Reply:
done that :)